public/gameInfo.php

```diff
@@ -1295,6 +1296,9 @@ function ResetProgress() {
                             echo " <span class='TrueRatio'>($achTrueRatio)</span>";
                             echo "</div>";
                             echo "<div class='mb-2'>$achDesc</div>";
+                            if ($flags != $officialFlag && isset($user) && $permissions >= Permissions::JuniorDeveloper) {
+                                echo "<p class='text-2xs'>Author: <a href='/user/$achAuthor/'>$achAuthor</a></p>";
+                            }
```

Comment:
Why not use `userAvatar`? These links won't have hover tooltips, whereas the author links at the top of the page do.